doc/userguide/rules/intro.rst

```diff
@@ -100,21 +102,20 @@ you can pick from. These are:
 * sip
 * http2
 
-The availability of these protocols depends on whether the protocol is enabled in the configuration file suricata.yaml.
+The availability of these protocols depends on whether the protocol is enabled in the configuration file, suricata.yaml.
```

Comment:
We are trying to enforce the <80 character limit for new changes, to comply with Sphinx recommendations. Can you make sure that the lines where you add changes are compliant?

 (this isn't a hard rule, I sometimes and up with 80 or 81)